src/test/java/Homework16.java

```diff
@@ -0,0 +1,44 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+
+public class Homework16 extends BaseTest {
+
+    @Test
+    public void Registration(){
+
+        //REGISTRATION TEST CASE :
+
+        //Preconditions: Chrome Browser should be open
+        //Step1: Open to 'Koel' homepage.
+        //Step 2: Click on Registration
+        //Step 3: Validate the Registration Page.
+
+        //PRECONDITION:
+        WebDriver driver = new ChromeDriver();
+        driver.manage().timeouts().implicitlyWait (Duration.ofSeconds(10));
+
+        //STEP1 : 'KOEL' HOMEPAGE:
+        String url =" https://bbb.testpro.io/";
+       driver.get(url);
+
+        //STEP 2: CLICK ON REGISTRATION LINK:
+        WebElement registrationField = driver.findElement(By.cssSelector("[id=\'hel\']"));
+        registrationField.click();
+
+        //STEP 3: VALIDATE REGISTRATION PAGE:
+
+        WebElement registerField = driver.findElement(By.cssSelector("[id=\'button\']"));
+        Assert.assertTrue(registerField.isDisplayed());
+
+
+
+
+
```

Comment:
**Please delete the empty line!** «An Empty Line is a Code Smell» 

The short answer is, They’re more an annoyance than a problem.They won’t actually break anything—which is why your code continues to run—but they make it awkward to read and change your code, especially if you’re dealing with multiple line breaks per line of real code. They also add just a tiny bit of size to your files, which is nice to reduce.